### DISCUSSION TITLE
Official Docker images available on any registry yet?

### DISCUSSION THREAD

**Discussion by capi** [TriliumNext/Notes#298] [Q&A / Support] [answered]
With the release of 0.90.3 (congrats!), is a docker image already available on a public registry? Or should I currently build them myself?

**Comment by meichthys** [comment, marked as answer]
https://hub.docker.com/r/triliumnext/notes

**Reply by capi** [reply]
Ah sure. Obvious if you think about it. I somehow couldn't find the link, maybe you can add it to the readme to make it official? Too many name-squatters nowadays, so name alone does not make it official.

**Reply by capi** [reply]
Well, I now found it on the docs page for server installation. Sorry. Did not dig deep enough.
